### ISSUE TITLE
The accuracy of the pretrained model

### ISSUE THREAD

**Issue by chn52979** [iMoonLab/PVRNet#10]
Hi, I validate the performance of the network with the pretrained model(PVNet2-ALEXNET-v94-ckpt.pth). However, I only get 92.55% accuracy. Have you tried to get the results with a higher version of pytorch？
Here is my environment: pytorch1.6.0 cuda10.1  NVIDIA TITAN Xp.
